access/advanced/permit.md

```diff
@@ -28,21 +28,15 @@ def alter(
 &nbsp;
 
 {: .experimental }
-> _Only a subset of available **Traits** are considered **Stable**. See [Traits](../segments_traits_operators#traits) for more details._
```

Comment:
This should be a link to supported traits, no? The "Options" link is also broken, which leads me to believe this was probably a mistake.

Reply:
yes, the "options" link above should have just been discussing traits as in connection admin. I fixed this as well as both links in permit.md

Reply:
This issue with the links was a broader problem and has been corrected in additional locations as well.

Reply:
This link is still broken. 

This should work:

```markdown
{: .experimental }
> _Only a subset of available **Traits** are considered **Stable**. See [Traits](../segments_traits_operators#traits) for more details._
```

Reply:
Link now works.